File: Day 7/Task2Multilevel/College.java
package Task2Multilevel;

public class College {
    String collegeName;
    String CollegePlace;
    int CollegeID;

    

    public College(String collegeName, String collegePlace, int collegeID) {
        this.collegeName = collegeName;
        CollegePlace = collegePlace;
        CollegeID = collegeID;
    }



    public void showCollegeDetails(){
        System.out.println(" ");
        System.out.println("College Name: "+collegeName);
        System.out.println("College Place: "+CollegePlace);
        System.out.println("College ID: "+CollegeID);
    }
}

File: day18new/college.kt
open class college {

    var collegeName:String="VPKBIET"
    var collegePlace:String="MIDC"

    inner class branch{
        var branchName:String="IT"
        var branchPlace:String="Delhi"
    }
    inner class student{
        var studentName:String="Yash"
        var studentPlace:String="Mumbai"
    }
}

fun main(){
    var clg=college();
    var output:String=clg.student().studentName;
    var branchoutput:String=clg.branch().branchName;
    println("Student name is $output & Branch is $branchoutput ")

}